src/lib.rs

```diff
@@ -705,6 +713,33 @@ fn json_obj_len(ctx: &Context, args: Vec<String>) -> RedisResult {
     json_len(ctx, args, |doc, path| doc.obj_len(path))
 }
 
+///
+/// JSON.CLEAR <key> [path ...]
+///
+fn json_clear(ctx: &Context, args: Vec<String>) -> RedisResult {
+    let mut args = args.into_iter().skip(1);
+    let key = args.next_string()?;
+    let arg = get_optional_arg(&mut args, "$");
+    let mut paths: Vec<Path> = vec![];
+    let path = backwards_compat_path(arg);
+    paths.push(Path::new(path));
+    while let Ok(mut arg) = args.next_string() {
```

Comment:
@oshadmi why do you need multi paths support? with the support of full JSONPath I don't see the need.

Reply:
@gkorland Do you mean that since multi paths could be expressed using a single JSONPath with wildcard, union iterator, recursive descent or slices?
It could still be useful to allow specifying several paths, either when they are plain/explicit, or when a complex path, with wildcards, etc., might yield unwanted paths, e.g., `$..attr` is too "permissive" when user only wants 2 specific paths out of many others, so `$.sub1.attr` and `$.sub2.attr` could be easily specified, while finding the exact single complex path could be harder.

Reply:
yes single "path" can be a union of multi paths.
And it's consistent with:  ```JSON.DEL <key> [path]```

Reply:
@gkorland 
The current [doc](https://oss.redislabs.com/redisjson/commands/#jsonget) specifies one or more `path` args **only** for `JSON.GET` (using three dots)
```
JSON.GET <key>
         [INDENT indentation-string]
         [NEWLINE line-break-string]
         [SPACE space-string]
         [NOESCAPE]
         [path ...]
```
But for [JSON.DEL](https://oss.redislabs.com/redisjson/commands/#jsondel) only a single path is specified:
`JSON.DEL <key> [path]`

For `JSON.CLEAR` we aimed to eventually support multi-paths, right?
So it would eventually be documented with three dots as:
`JSON.CLEAR <key> [path ...]`
With the return value being the number of paths cleared, similar to `JSON.DEL`.

Additional question (also related to single path which could use wildcards, union iterator, etc. to match several elements):
What should be done if one or more paths do not exist or do not resolve to an array or an object:
1) Should the command be atomic, and avoid doing just part of the job? Should it then return the error messages reporting 
 the wrong paths, or just do nothing or return nil?
Or
2) Should the command report on erroneous path and resume with the rest of the paths, clearing only existing and matching arrays/objects and skipping the rest? (if the command is not atomic, the return value would be an array of numbers and/or error messages?)